core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java

```diff
@@ -510,8 +514,7 @@ public void index(FullIndexEntry indexEntry, String key, String fieldName, TextF
 
   public String duplicateContent(Field field, FieldTemplate template, ForeignPK from, ForeignPK to,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "field". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1172)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Remove this use of "ForeignPK"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Remove this use of "ForeignPK"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)